config/bidderinfo_test.go

```diff
@@ -518,6 +518,7 @@ func TestBidderInfoValidationNegative(t *testing.T) {
 			[]error{
 				errors.New("bidder: bidderB cannot be an alias of an alias: bidderA"),
 				errors.New("bidder: bidderC not found for an alias: bidderB"),
+				errors.New("bidder: bidderC not found for an alias: bidderB"),
```

Comment:
Is there now two errors for the alias not found error?

Reply:
yes, one from `validateAliases` and another from `validateAliasCapabilities`